manifests/s/SophosLimited/SophosSetup/1.7.6.0/SophosLimited.SophosSetup.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SophosLimited.SophosSetup
+PackageVersion: 1.7.6.0
+InstallerSwitches:
+  Silent: h
+  SilentWithProgress: Silent
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://shareext.blob.core.windows.net/mis/SophosSetup.exe

```

Comment:
This doesn't look like an official Sophos URL
Also, the installer doesn't seem to exit after a successful install